applications/nrf_desktop/src/modules/Kconfig.dvfs

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_DVFS
+	bool "DVFS module"
+	depends on SOC_NRF54H20_CPUAPP
+	depends on CAF_BLE_COMMON_EVENTS || (BT_MAX_CONN = 1)
```

Comment:
`depends on !CAF_BLE_COMMON_EVENTS || (BT_MAX_CONN = 1)` (as suggested in https://github.com/nrfconnect/sdk-nrf/pull/16492#discussion_r1687660293)